app/stores/DocumentsStore.ts

```diff
@@ -573,9 +573,9 @@ export default class DocumentsStore extends BaseStore<Document> {
   duplicate = async (document: Document): Promise<Document> => {
     const append = " (duplicate)";
     const res = await client.post("/documents.create", {
-      publish: !!document.publishedAt,
-      parentDocumentId: document.parentDocumentId,
-      collectionId: document.collectionId,
+      publish: document.template ? true : false,
+      parentDocumentId: null,
+      collectionId: document.template ? document.collectionId : null,
```

Comment:
```suggestion
      publish: document.isTemplate ? true : false,
      parentDocumentId: null,
      collectionId: document.isTemplate ? document.collectionId : null,
```

Prefer isTemplate for clarity, need to look into why we have both 😆